usecases/backup/scheduler.go

```diff
@@ -228,9 +227,9 @@ func (s *Scheduler) BackupStatus(ctx context.Context, principal *models.Principa
 	defer func(begin time.Time) {
 		logOperation(s.logger, "backup_status", backupID, backend, begin, err)
 	}(time.Now())
-	if err := s.authorizer.Authorize(principal, authorization.READ, authorization.Backups(backend)); err != nil {
-		return nil, err
-	}
+	// if err := s.authorizer.Authorize(principal, authorization.READ, authorization.Backups(backend)); err != nil {
+	// 	return nil, err
+	// }
```

Comment:
**Suggestion:** If we leave this commented-out code in, can we add a comment above for why we aren't authorizing this request?